src/EventListener/RenderingForwarder.php

```diff
@@ -70,28 +69,35 @@ public function registerTemplates(): void
     /**
      * @Hook("parseTemplate")
```

Comment:
Maybe defining a very low priority might makes sense so that other listeners are called before

Reply:
That's also a good point :smile: